.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Vividus App CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install npm dependencies
+        run: |
+          npm install
```

Comment:
add super lineter? :)

Reply:
It's worth to try in the future :)

Reply:
imho js based app is a best choice to use it asap

Reply:
I meant files like json etc, superlinter already uses eslint for js based apps